views/index.handlebars

```diff
@@ -1,10 +1,36 @@
-<h1>{{ h1 }}</h1>
+<div id="topheader">
+	<h1 class="siteheader"> Pro Rubrics </h1>
```

Comment:
siteheader is not camelCased. Instead of using a class here could we not just target `header.top h1` assuming some changes were made to the topheader class?